exetera/core/readerwriter.py

```diff
@@ -410,15 +410,21 @@ def write(self, values):
 
 class NumericImporter:
     def __init__(self, datastore, group, name, nformat, parser, invalid_value=0,
+                 validation_mode='allow_empty', create_flag_field=True, flag_field_suffix='_valid',
                  timestamp=None, write_mode='write'):
         if timestamp is None:
             timestamp = datastore.timestamp
         self.data_writer = NumericWriter(datastore, group, name,
                                          nformat, timestamp, write_mode)
-        self.flag_writer = NumericWriter(datastore, group, f"{name}_valid",
-                                         'bool', timestamp, write_mode)
+        
+        self.flag_writer = NumericWriter(datastore, group, f"{name}{flag_field_suffix}",
```

Comment:
This line is too long if you're having to use the newline escape. The infix if-else expression should really only be used for small expressions, so this I would break out into an if statement:

```python
self.flag_writer = None
if create_flag_field 
    self.flag_writer = NumericWriter(datastore, group, f"{name}{flag_field_suffix}", 'bool', timestamp, write_mode)
```